Appraisals

```diff
@@ -38,3 +38,13 @@ appraise "rails6.0" do
   gem "spring-watcher-listen", "~> 2.0.0"
   gem "sqlite3", "~> 1.4"
 end
+
+appraise "rails6.1" do
+  gem "byebug"
+  gem "listen", "~> 3.2"
+  gem "puma", "~> 4.1"
```

Comment:
Looks like the [default Puma version](https://github.com/rails/rails/blob/v6.1.1/railties/lib/rails/generators/app_base.rb#L209) for a new Rails app is "~> 5.0", so we will probably need to match that to get the tests passing (basically we want the versions here in the appraisal to match whatever is in the Gemfile of a brand new Rails app).

```suggestion
  gem "puma", "~> 5.0"
```

Reply:
Makes sense, thanks for the review and spotting this! Resolved.